toolkit.lisp

```diff
@@ -13,6 +13,17 @@
 (defun unix->universal (unix)
   (+ unix *unix-epoch-difference*))
 
+(defun universal->timestring (universal)
+  (multiple-value-bind (seconds minutes hours day month year x daylight-saving timezone)
+      (decode-universal-time universal)
```

Comment:
Instead of dealing with the timezone and dst return values, just pass 0 as the timezone argument to `decode-universal-time`.